lib/minitest.rb

```diff
@@ -19,8 +19,9 @@ module Minitest
   ##
   # Parallel test executor
 
-  mc.send :attr_accessor, :parallel_executor
-  self.parallel_executor = Parallel::Executor.new((ENV['N'] || 2).to_i)
+  def self.parallel_executor
```

Comment:
I don't mind memorizing the attribute, I usually prefer it. But we still need to be able to set the attribute even when memorizing it.

```ruby
mc.send :attr_writer, :parallel_executor
def self.parallel_executor
  @parallel_executor ||= Parallel::Executor.new((ENV['N'] || 2).to_i)
end
```